decorator.rb

```diff
@@ -0,0 +1,9 @@
+class Decorator
+  def initialize(nameable)
+    @nameable = nameable
+  end
+
+  def correct_name
+    @nameable.correct_name
+  end
+end
```

Comment:
- [Optional] I think it would be better if you import the `nameable` class in this file and ensure that the base decorator class inherits from the nameable class. Currently, when I run your app I experience errors because of missing imports in some files. Please see the example code below for an example of how to achieve this. 

## Example code
> I highly recommend this change because some features are not working in your App

```ruby
require_relative 'nameable_class'

class Decorator < Nameable
  def initialize(nameable)
    @nameable = nameable
  end

  def correct_name
    @nameable.correct_name
  end
end
```

## Project requirements
![Screenshot from 2023-08-10 01-11-01](https://github.com/kayjeee/SchoolLibraryMicroverse/assets/16392046/ad881771-0a94-410f-be7e-e6142af0800c)

Reply:
done